Add info on ASCII module

tledoux opened the pull request:
Document withTextMD parameter.
Describe the representation information output.

tledoux committed:
Add info on ASCII module

Document withTextMD parameter.
Describe the representation information output.

carlwilson commented:
Hi @tledoux it appears there are broken link issues: https://travis-ci.org/openpreserve/jhove/builds/226402556#L421

tledoux committed:
Correct broken link

The standard representation information is documented in the general documentation

carlwilson committed:
Merge branch 'gh-pages' into tledoux-module-doc
